mango-python/bdgenomics/mango/variantViz.py

```diff
@@ -0,0 +1,192 @@
+#
```

Comment:
Why do we have a file called "variants.py" and variantViz.py"? The both have visualizations.